docs/coordinates/index.rst

```diff
@@ -338,7 +338,7 @@ To see the list of site names available, use
 :func:`~astropy.coordinates.EarthLocation.get_site_names`::
 
     >>> EarthLocation.get_site_names()  # doctest: +REMOTE_DATA
-    ['ALMA', 'AO', 'ARCA', 'ASKAP', 'ATST', 'Anderson Mesa', ...]
+    ['ALMA', 'AO', 'ARCA', 'ASKAP', 'ATA', ATST', ...]
```

Comment:
We can reduce the chances of the expected output changing if we write out less names:
```suggestion
    ['ALMA', 'AO', 'ARCA', ...]
```